dfm-sideline-sidekick-app/pages/GlobalSearch.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import { FlatList, Text, TextInput, View } from "react-native";
+import Icon from "react-native-vector-icons/FontAwesome"; 
+
+import styles from "./GlobalSearchStyles";
+
+type Document = {
+  id: string;
+  title: string;
+  subtitle: string;
+};
+
+const documents: Document[] = [
+  { id: "1", title: "Cervical Spine Injury", subtitle: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do." },
+  { id: "2", title: "Cervical Strain", subtitle: "Lorem ipsum dolor sit amet, consectetur adipiscing." },
+  { id: "3", title: "Stroke", subtitle: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor." },
+];
+
+const SearchBarComponent = () => {
+  const [query, setQuery] = useState("");
+  const [filteredDocuments, setFilteredDocuments] = useState<Document[]>([]);
+
+  const handleSearch = (text: string) => {
```

Comment:
For the future, maybe extract the handleSearch functionality to a separate file, as this will gain complexity when having to search through device storage